Product_Landing_Page/public/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Landing Page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="../css/style.css">
+  <link rel="stylesheet" type="text/css" href="../css/font-awesome.min.css">
+</head>
+<body>
+  <div class="container" id="container">
+  <header class="header" id="header">
+    <div class="page-header" id="page-header">
+
+    </div>
+    <div class="page-logo" id="page-logo">
+      <img src="../images/original_trombones.png" alt="Logo" id="logo-image">
+    </div>
+    <div class="navigational-bar" id="navigational-bar">
+      <ul>
+        <li><a class="navigational-link" href="#">Features</a></li>
+      </ul>
+      <ul>
+        <li><a class="navigational-link" href="#">How It Works</a></li>
+      </ul>
+      <ul>
+        <li><a class="navigational-link" href="#">Pricing</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <div class="under-header-section" id="emailform">
+    <h2>Handcrafted, home-made masterpieces</h2>
+    <form class="form" id="form"  method="post">
+
+      <input type="email" name="email" id="email" placeholder="Enter your email address" required>
+      <input type="submit" name="submit" id="submit" value="Get Started" >
+    </form>
+  </div>
+  <div class="features-section">
+
+    <div class="feature">
+      <div class="icons">
+        <i class="fas fa-clock" style="font-size:60px;color:darkorange;"></i>
+      </div>
```

Comment:
remove the inline styling. And convert this into class.

Reply:
Done